closed/adds/jdk/src/solaris/native/jdk/crypto/jniprovider/NativeCrypto_md.c

```diff
@@ -29,13 +29,18 @@
 #include <dlfcn.h>
 #include "NativeCrypto_md.h"
 
+#if defined(UNIX)
```

Comment:
As mentioned elsewhere, this should be `defined(__linux__)`

Reply:
Edited.